ambry-api/src/main/java/com/github/ambry/config/MySqlNamedBlobDbConfig.java

```diff
@@ -19,6 +19,7 @@ public class MySqlNamedBlobDbConfig {
   private static final String PREFIX = "mysql.named.blob.";
   public static final String DB_INFO = PREFIX + "db.info";
   public static final String DB_TRANSITION = PREFIX + "db.transition";
+  public static final String DB_NEW_TABLE = PREFIX + "db.newtable";
```

Comment:
can you rename this to something more descriptive, something like `db.read.from.new.table`.

Reply:
Done.